Sources/ARSAL_Time.c

```diff
@@ -46,7 +63,31 @@ int ARSAL_Time_GetTime (struct timespec *res)
     }
     /* No else --> Args check (return -1) */
 
-#if defined(HAVE_CLOCK_GETTIME) && defined (HAVE_DECL_CLOCK_MONOTONIC)
+#if defined(HAVE_WINDOWS_H)
+
+    static LARGE_INTEGER perf_freq = { .QuadPart = 0 };
+    static LARGE_INTEGER start_point = { .QuadPart = 0 };
+
+    result = 0;
+    if (perf_freq.QuadPart == 0) {
```

Comment:
I think there is a possible race here if 2 thread tries to call this func.
I thread A is interrupted between the call to QueryPerformanceFrequency and QueryPerformanceCounter, 
ThreadB will read a valid freq but an not yet set start point

You could use this implementation (done in one of our other lib):
https://github.com/Parrot-Developers/libfutils/blob/master/src/timetools.c#L44